src/main/java/esi/backend/service/CustomerService.java

```diff
@@ -0,0 +1,52 @@
+package esi.backend.service;
+
+import esi.backend.exception.ResourceNotFoundException;
+import esi.backend.exception.UnauthorizedException;
+import esi.backend.model.Car;
+import esi.backend.model.Customer;
+import esi.backend.model.Request;
+import esi.backend.repository.CustomerRepository;
+import esi.backend.security.service.UserDetailsImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+@Service
+public class CustomerService {
+
+    @Autowired
+    CustomerRepository customerRepository;
+
+    public List<Customer> getAllCustomers(){
+        List<Customer> customers = new ArrayList<>();
+        customerRepository.findAll().forEach(customers::add);
+        return customers;
+    }
```

Comment:
Have `ResponseEntity` as the return type of all functions in all `Service` classes.